### PR TITLE
Scripts to check issue 1831

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check [#1831](https://github.com/smartdevicelink/sdl_core/issues/1831)

This PR is **not ready** for review.
This is **draft** version of the script.

### Summary
A test script to verify that issue is fixed

### ATF version
[develop](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Local Variables ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua]: for navi app isMediaApplication is false

File: test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerAppWOPTU, { 1 })
+runner.Step("Activate App", common.activateApp)
+
+-- [[ Test ]]
+runner.Title("Test")
+runner.Step("Application disconnect", closeSession)
+runner.Step("Open session", openConnection)
+runner.Step("Clean session", cleanSessions)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua]: cleanSessions must be executed before  openConnection and after closeSession

File: test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Local Variables ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Local Functions ]]
+local function closeSession()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua]: in function is performed the connection closing, not session, please update function name

File: test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+	common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = true })
+	test.mobileConnection:Close()
+end
+
+local function openConnection()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1831_Crash_during_Navigation_app_resume.lua]: create connection can be define like 
```
local function openConnection()
  test.mobileConnection:Connect()
  EXPECT_EVENT(events.connectedEvent, "Connected")
  :Do(function()
      utils.cprint(35, "Mobile connected")
    end)
end
```

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by GetmanetsIrina** [comment]
Resumption of Navi application is covered by https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/SDL5_0/ConditionalResumption/003_conditional_resumption_specific_hmi_level_navi.lua and also some other scripts.
So PR is closed.

**Pull request closed without merging**